CrossPlatformDesktopProject/CrossPlatformDesktopProject/GameStates/ThunderDomeState.cs

```diff
@@ -96,9 +118,9 @@ private void GenerateNextWave()
         enemyWaves.Add(new Skeleton(coords[0], coords[1], game));
 
         waveNumber += 1;
-        if(waveNumber > enemyWaves.Count)
+        if (waveNumber >= enemyWaves.Count)
```

Comment:
Think this is the culprit - change it to > instead of >= and it should work as expected?